code/modules/customitems/item_spawning.dm

```diff
@@ -190,7 +190,7 @@
 
 		// Check for required access.
 		var/obj/item/weapon/card/id/current_id = M.wear_id
-		if(citem.req_access && citem.req_access > 0)
+		if(citem.req_access && citem.req_access > 0) // These are numbers, not lists
```

Comment:
is the first clause really necessary? 0 and null are both false-y values, and so evaluate to 0, which should fail the second clause.

Reply:
True, though not code I added or changed